apps/builder/app/builder/features/style-panel/sections/sections.ts

```diff
@@ -25,6 +25,7 @@ export const sections = new Map<
     Section: (props: SectionProps) => ReactNode;
   }
 >([
+  ["transitions", transitions],
```

Comment:
Will remove this change, once adding tests. Just added for dev time convenience.